src/api/loginApi.js

```diff
@@ -0,0 +1,24 @@
+async function fetchFromServer(auth_code, url) {
+    const requestOptions = {
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: auth_code,
+    };
+    const data = await fetch(
+        `${process.env.REACT_APP_API_ROOT}/${url}`,
+        requestOptions,
+    );
+    return data;
+}
+export async function signInUser(auth_code) {
+    auth_code = JSON.stringify(auth_code);
```

Comment:
Nothing wrong here, but I'd suggest creating and using a new auth_code variable here instead of reassigning to the same one